pkgs/top-level/all-packages.nix

```diff
@@ -23041,7 +23041,13 @@ with pkgs;
 
   onlyoffice-documentserver = callPackage ../servers/onlyoffice-documentserver { };
 
-  outline = callPackage ../servers/web-apps/outline { };
+  outline = callPackage ../servers/web-apps/outline (
```

Comment:
I'm not sure why you need to do all this, why not just `nodejs = nodejs-16_x;` (I think we are in a `rec`) ?

Reply:
(because you want to enable further overrides of the Node version? I am not sure why further .override are no good?)

Reply:
Oh, yeah, I can drop `self` all together.